README.md

```diff
@@ -23,6 +23,8 @@ If you want to just jump in, go straight to the demo. The demo uses [Ranklib](ht
 
 See the full list of [prebuilt versions](http://es-learn-to-rank.labs.o19s.com). If you don't see a version available, see the link below for building or file a request via [issues](https://github.com/o19s/elasticsearch-learning-to-rank/issues).
 
+**IMPORTANT**: please be sure to read the [known issues](KNOWN_ISSUES.md) before installing this plugin.
```

Comment:
I would probably not be so strong, and make an own section, something like



```
**Know issues**
As any other peace of software, this plugin is not except of issues. Please read  the [known issues](KNOWN_ISSUES.md) to know more about the current issues that we are aware of. This file might include workarounds to mitigate them when possible.

```
what do you think?

Reply:
sure :)
(even if this bug is particularly serious)

Reply:
is serious and important, but I found the statement in the readme too serious 😄 .... i kinda like the way the known_issues section is written in the elasticserach docs, important bugs, communicated nicely.